sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuSortExec.scala

```diff
@@ -431,8 +410,6 @@ case class GpuOutOfCoreSortIterator(
         while(sortedIt.hasNext) {
           val sortedTbl = sortedIt.next()
           val rows = closeOnExcept(sortedTbl) { _ =>
```

Comment:
The closeOnExcept is now overkill, can just be rows = sortedTbl.getRowCuont.toInt.